From 6349dd9d97ce04219383ca181ae16107e5953785 Mon Sep 17 00:00:00 2001
From: Peter Harliman Liem <peter.harliman.liem@intel.com>
Date: Mon, 4 Mar 2019 13:12:14 +0800
Subject: [PATCH] DRVLIB_SW-2236 - Fix bug in gphy fused rcal/rcm calculation

- fused rcal value is wrongly never set

These will only affect gphy with fused rcal/rcm values.
---
 drivers/net/ethernet/lantiq/xrx500_phy_fw.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/lantiq/xrx500_phy_fw.c b/drivers/net/ethernet/lantiq/xrx500_phy_fw.c
index 70cfab8da1e2..b1ac073beaee 100644
--- a/drivers/net/ethernet/lantiq/xrx500_phy_fw.c
+++ b/drivers/net/ethernet/lantiq/xrx500_phy_fw.c
@@ -204,6 +204,8 @@ static u32 prx300_gphy_config_rcal_rcm(struct xway_gphy_data *priv)
 	if (!val)
 		goto exit;
 
+	rcal = val;
+
 	/* rcm calculation routine */
 
 	/* enable block access */
